airbyte-integrations/connectors/source-airtable/source_airtable/auth.py

```diff
@@ -42,8 +44,15 @@ def _get_refresh_access_token_response(self) -> Mapping[str, Any]:
             data=self.build_refresh_request_body(),
             headers=self.build_refresh_request_headers(),
         )
+        content = response.json()
+        if response.status_code == 400 and content.get("error") == "invalid_grant":
+            raise AirbyteTracedException(
+                internal_message=content.get("error_description"),
+                message="Refresh token is invalid. Please update it in the config.",
```

Comment:
We need to provide understandable user messages, users click 'Authenticate' button in set up page, which doesn't show refresh token field. if we want to ask user create a new one we should ask him to 'Re-authenticate'. 

`Refresh token is invalid/expired. Please re-authenticate to restore access to Airtable.` - more suitable error message.

Reply:
Thanks for the advice, updated